lib/unleash/toggle_fetcher.rb

```diff
@@ -82,6 +82,8 @@ def update_engine_state!(toggle_data)
 
       # notify all threads waiting for this resource to no longer wait
       self.toggle_resource.broadcast
+    rescue StandardError => e
```

Comment:
This causes the SDK to eat this error without logging anything so this change gives us some insight into yggdrasil failures